tools/rpk-docs/rpk-docs-handler.js

```
@@ -2891,6 +2891,7 @@ function runValidation(outputDir, options = {}) {
 
 module.exports = {
   handleRpkDocsGeneration,
+  prepareSourceFromRef,
```

Biome (2.5.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)